dbt_platform_helper/templates/environments/main.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  environments = yamldecode(file("../../../platform-config.yml"))["environments"]
+  env_config   = { for name, config in local.environments : name => merge(lookup(local.environments, "*", {}), config) }
+  args = {
+    application    = "{{ application }}"
+    services       = yamldecode(file("../../../platform-config.yml"))["extensions"]
+    dns_account_id = local.env_config["{{ environment }}"]["accounts"]["dns"]["id"]
+  }
+}
+
+terraform {
+  required_version = "~> 1.8"
+  backend "s3" {
+    bucket         = "terraform-platform-state-{{ config.accounts.deploy.name }}"
```

Comment:
```suggestion
    profile        = "{{ aws_profile }}"
    bucket         = "terraform-platform-state-{{ config.accounts.deploy.name }}"
```

Reply:
Is this needed? I don't see it in any of the envs on main.

Reply:
@WillGibson ?